eventApp/app/src/main/java/org/lightsys/eventApp/tools/qr/launchQRScanner.java

```diff
@@ -111,4 +119,12 @@ private void initiatePopupWindow() {
         dialog.show();
     }
 
+    @Override
+    protected void onDestroy(){
+        try {
+            super.onDestroy();
+        } catch (Exception e) {
+            super.finishAndRemoveTask();
```

Comment:
It would be good to have a comment here describing what this apparently accomplishes.